setup.py

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+import os
+
+engine_dir = os.path.join(os.path.dirname(__file__), "robotick-engine")
+result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)